src/GitHub.Api/NewTaskSystem/ProcessTask.cs

```diff
@@ -134,14 +173,37 @@ public void Run()
             }
 
             if (Process.StartInfo.RedirectStandardOutput)
-                Process.BeginOutputReadLine();
+            {
+                outputReset.Reset();
+                Process.StandardOutput.BaseStream.BeginRead(
```

Comment:
If you read the standard output using a loop there, it blocks processing.
Because `RunCredentialHelper` writes to the standard input of `git-credential-wincred` command when signing in to GitHub.
https://github.com/github-for-unity/Unity/blob/1476044c625f615e74b282ecb5c4c286fe0bb86b/src/GitHub.Api/Git/GitCredentialManager.cs#L147-L172

For that reason, I think that we need an asynchronous `BeginRead`.
Please tell me if there is a better way.